docs/routing/special_routes.md

```diff
@@ -0,0 +1,62 @@
+# Special routes
+
+To see special routes, use the `Show special` switcher at the top of the routes page
+
+It is used to render errors, if an application with the `Primary` or `Essential` type falls on the page, ILC will not find a match between the address bar and `Route`, `Template` will not be specified in the route chain, etc., ILC will use `Special route` or simple template with an error, depending on the situation:
+
+## '404' Error
+
+![ILC registry special routes switcher](../assets/routes/special-routes-switcher.png)
+
+- This error usually means that the requested resource was not found on the server.
+
+- When using `Special route` all standard routes are ignored, including `*` with any `Order pos`.
```

Comment:
I got what do you want to say here.
But I would say something like `Unlike simple route which uses composition, the special route has own a full configuration. So if you declared, for example, some header by default for any route and want to see it at 404 special route, then you should declare header personally for 404 special route too.`